src/main/kotlin/com/atlassian/performance/tools/report/api/NonparametricFacade.kt

```diff
@@ -0,0 +1,58 @@
+package com.atlassian.performance.tools.report.api
+
+import com.numericalmethod.suanshu.stats.test.rank.wilcoxon.WilcoxonRankSum
+import org.apache.commons.math3.stat.descriptive.rank.Median
+import org.apache.commons.math3.stat.inference.KolmogorovSmirnovTest
+import java.lang.Exception
+
+class NonparametricFacade(
```

Comment:
This is a very vague name.
"Non-parametric" - ok, that's an interesting part.
"Facade" - we're hiding some things, but so does every other class.

However, there are a lot of non-parametric logic out there. How does it differ from Kolmogorov-Smirnov test for example?
We can call it the Natkovskyi test 🙂

Reply:
Facade came after the Facade pattern when I was thinking of hiding more complex stuff behind simpler methods. But I agree, it's not a good name. Could you suggest a better one? I'm not very good at that.

Reply:
What is the responsibility of this class? We can infer that from its public API. It does something with:
* baseline and experiment
* alpha levels for MW and KS
* shifts
* regressions
* distributions

Sounds like some kind of test. `ShiftedDistributionRegressionTest`?
Perhaps we should clean up its public members to use more consistent vocabulary.
Alternatively, we can simplify to `NatkovskyiShiftTest` and list the characteristics in the KDoc.